docs/client_server_mode.md

```diff
@@ -20,6 +20,12 @@ Start SAI Challenger server:
 ./run.sh -i server -a trident2 -t saivs
 ```
 
+If you are using server docker on the same host you can run the following script for creating links between client and server dockers:
```

Comment:
Please correct me if I'm wrong, but with the changes proposed in this PR, the client-server mode will not work without running `./veth-create-host.sh` because syncd will wait for the ports infinitely. Which does not make sense if we are using libsaivs because it does not support traffic running anyway.
Can we have this as an optional way to go?

Reply:
It does make sense even with saivs since you can run traffic to/from the host. Actually, I tested all that staff using saivs.
This code will be helpful for companies that want to replace saivs with their own simulator.

Reply:
I do understand that this was tested with saivs. But that looks as an optional step, not mandatory. The TCs will still fail with `--traffic` option. So, we enforce the user to always run this step even it's optional. From the other hand, the client-serve mode is rarely used with libsaivs... Let me think about possible alternatives and get back to you here.

Note, in case someone wants to run ASIC simulator instead, he should create a separate supervisord.conf with simulator entry inside. E.g., https://github.com/opencomputeproject/SAI-Challenger/blob/561a51a645feefc7babe7c0a484b384c4b60e9b4/platform/intel/tofino/model/configs/supervisord.conf#L64

Reply:
Please update the description that this step is mandatory and let's move forward.
I think the description can be something as follows:
```
Create traffic links between SAI Challenger server and client:
'''sh
bash -c ./veth-create-host.sh  <server docker>  <client docker>
'''
```

Reply:
I'll update this.
IMO if a user does not need traffic, there is no reason to have client/server mode.

Reply:
ok